src/assets/locks/fingerprint_lock/fingerprint_lock.asset.ts

```diff
@@ -0,0 +1,21 @@
+DefineLockAsset({
+	name: 'Fingerprint Lock',
+	assetPreferenceDefault: 'maybe',
```

Comment:
This lock is even stricter than the combination lock, which is set to 'prevent', so this one should be as well, by default.
```suggestion
	assetPreferenceDefault: 'prevent',
```